doc/tech/approach.md

```diff
@@ -3,97 +3,226 @@
+
+SDNo controllers can request Underlay Networking encapsulation and mapping to be done by signaling to an SDNu controller. There are however today no standardized way for this signaling and by that there is a missing reference point and API description in this architecture. 
+
+Multiple instances of Container as a Service (CaaS) Virtualization Layers running on an Infrastructure as a Service (IaaS) Virtualization Layer could make use of the IaaS layer to handle the required Underlay Networking separation. In these cases, the IaaS Virtualisation Infrastructure Manager (VIM) could include a SDNu control interface enabling automation.
+
+> **Note:** The Reference Model describes a logical separation of SDNu and SDNo interfaces to clarify the separation of administrative domains where applicable. In real deployment cases an Operator can select to deploy a single SDN controller instance that implements all needed administrative domain separations or have separate SDN controllers for each administrative domain. A common deployment scenario today is to use a single SDN controller handling both Underlay and Overlay Networking which works on the implementation level when there is only one administrative domain that owns both the HW Infrastructure and the only  SW Virtualisation Infrastructure. The nature of a shared Underlay Network that shall ensure separation and be robust is that all code in the forwarding plane and in the control plane must be under the scrutiny and life cycle management of the HW Infrastructure Layer.
+One consequence of this is that the Reference Architectures must not model collapsed SDNo and SDNu controllers since they must stay unaware of other deployed implementations  running on the same HW Infrastructure.
+
+Hence, when Networking now gets introduced into the Reference Model, it is important that SDN also is considered and that the two now existing Reference Architectures are harmonised with the new common Networking Reference Model. In the absence of a Networking Reference Model, both Reference Architectures have mostly been specifying networking as if they where the only single SW Infrastructure on a dedicated HW resource pool.
+
+<a name="5.1.3.4"></a>
+#### 5.1.3.4 Programmable Networking Fabric concept
+
+The concept of a Programmable Networking Fabric pertains to the ability to have an effective forwarding pipeline (a.k.a. forwarding plane) that can be programmed and/or configured without any risk of disruption to the shared Underlay Networking that is involved with the reprogramming for the specific efficiency increase.
```

Comment:
Being configured without any risk of disruption should apply equally, whether the fabric is programmable or not. I would have thought in our context the programmable fabric would refer to the individual fabric elements' ability to have their forwarding logic explicitly controlled by a centralized executive, such that the collection of elements logically operate as a single, coordinated, highly scalable fabric. My 2 cents...

Reply:
I argue that a Programmable Fabric is something you could Configure and/or Program as a Fabric.
The difference in between Configuring and Programming is in this context not an exact science as I see it.
In the end it will also be up to the use cases of who has the administrative rights to install a new program (that could include new functionality, characteristics or just a SW Upgrade) and who has the rights to invoke it through configuration.
I think we are meaning the same but possibly express it a little bit differently.
I have take some of your feedback and updated the language a bit that hopefully makes it a bit clearer.